src/command_modules/azure-cli-advisor/azure/cli/command_modules/advisor/_params.py

```diff
@@ -0,0 +1,24 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+from azure.cli.core.commands import register_cli_argument
+
+register_cli_argument('advisor recommendation generate', 'timeout', options_list=('--timeout', '-t'), required=False, help='Timeout in seconds')
+
+register_cli_argument('advisor recommendation list', 'ids', nargs='+', options_list=('--ids'), required=False, help='One or more resource IDs (space delimited). If provided, no other "Resource Id" arguments should be specified.')
+register_cli_argument('advisor recommendation list', 'rg_name', options_list=('--resource-group', '-g'), required=False, help='Name of resource group.')
+register_cli_argument('advisor recommendation list', 'category', options_list=('--category', '-c'), required=False, help='Name of category.')
+
+register_cli_argument('advisor recommendation disable', 'ids', nargs='+', options_list=('--ids'), required=True, help='One or more resource IDs (space delimited). If provided, no other "Resource Id" arguments should be specified.')
+register_cli_argument('advisor recommendation disable', 'days', options_list=('--days', '-d'), required=False, help='Number of days to disable. If not specified, the recommendation is disabled forever.')
+
+register_cli_argument('advisor recommendation enable', 'ids', nargs='+', options_list=('--ids'), required=True, help='One or more resource IDs (space delimited). If provided, no other "Resource Id" arguments should be specified.')
+
+register_cli_argument('advisor configuration get', 'rg_name', options_list=('--resource-group', '-g'), required=False, help='Name of resource group.')
```

Comment:
Use `resource_group_name_type` in `azure.cli.core.commands.parameters`.
For  consistency with other modules, rename rg_name to resource_group_name.
i.e. https://github.com/Azure/azure-cli/blob/dev/src/command_modules/azure-cli-container/azure/cli/command_modules/container/_params.py#L36